nntrainer/graph/network_graph.cpp

```diff
@@ -402,12 +415,12 @@ int NetworkGraph::realizeMultiOutputType(Layer &current) {
 }
 
 int NetworkGraph::isCompilable() {
```

Comment:
How about adding here, a bool or some logic to check if compiled already and check it here to prevent compiling twice?

Reply:
Turned out that there is a bool `compiled` I wonder if that can be used here as well?

Reply:
The check was being done in `compile()` itself, but it's better suited here. Moved here.